packages/client/src/pages/home/action-group/post-dialog.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Dispatch, SetStateAction } from "react";
+import {
+  Button,
+  Checkbox,
+  createStyles,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  FormControlLabel,
+  makeStyles,
+  TextField,
+} from "@material-ui/core";
+import styled from "styled-components";
+import TagSelect from "./tag-select";
+
+const useStyles = makeStyles(() =>
```

Comment:
i think we're going to be using `styled-components` for custom styling but don't quote me on that, we'll prob finalize decision tmrw

Reply:
Yeah wasn't sure about using this, I'll change it after we decide, but I just wasn't sure how to use the styled-components for this part 😆